webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionRepositoryCell.js

```diff
@@ -36,31 +40,27 @@ const ContentViewVersionRepositoryCell = ({
     },
     docker_tag: {
       name: __('Container image tags'),
-      url: `docker_tags?repositoryId=${libraryInstanceId}`,
+      to: `dockerTags?library_instance_id=${libraryInstanceId}`,
     },
     erratum: {
       name: __('Errata'),
-      url: `errata?repositoryId=${libraryInstanceId}`,
+      to: `errata?library_instance_id=${libraryInstanceId}`,
     },
     file: {
       name: __('Files'),
-      url: `files?repositoryId=${libraryInstanceId}`,
+      to: `files?library_instance_id=${libraryInstanceId}`,
     },
     module_stream: {
       name: __('Module streams'),
-      url: `products/${id}/repositories/${libraryInstanceId}/content/module_streams`,
-    },
-    package: {
-      name: __('Packages'),
-      url: `products/${id}/repositories/${libraryInstanceId}/content/packages`,
+      to: `moduleStreams?library_instance_id=${libraryInstanceId}`,
     },
     package_group: {
       name: __('Package groups'),
-      url: `products/${id}/repositories/${libraryInstanceId}/content/package_groups`,
+      to: `rpmPackageGroups?library_instance_id=${libraryInstanceId}`,
     },
     rpm: {
-      name: __('Rpm packages'),
-      url: `packages?repositoryId=${libraryInstanceId}`,
+      name: __('RPM packages'),
+      to: `rpmPackages?library_instance_id=${libraryInstanceId}`,
     },
```

Comment:
This order should be changed to this to make it more consistent with elsewhere:
![Screenshot from 2022-02-14 13-27-05](https://user-images.githubusercontent.com/21146741/153924198-efb92833-9279-4d94-8a46-5caaa9d67d5a.png)

Reply:
Roger